packages/core/package.json

```diff
@@ -19,6 +20,8 @@
     "decimal.js": "^10.2.1"
   },
   "devDependencies": {
+    "@eth-optimism/smock": "^1.1.9",
+    "@nomiclabs/hardhat-ethers": "^2.0.2",
```

Comment:
See my prior note about `hardhat-ethers`, but also: do we need to import `hardhat-ethers` here if its imported by common and injected into the `HardhatConfig.js` file?

Reply:
good point. will test to double check.